core/src/type_info.rs

```diff
@@ -38,132 +38,151 @@ lazy_static! {
             "".to_string(),
             "TYPE_NAME".to_string(),
             BsonTypeInfo::STRING,
+            TypeMode::Standard,
```

Comment:
Yeah I think you are right. I'll make that change.

Reply:
Is there a better way to handle this other than by creating a duplicate METADATA vector for TypeMode::Simple? This would basically be what I did before we decided to get rid of all the duplicate code.

Reply:
I might be misunderstanding here, but why would there be two types here? This is describing column metadata, which is statically known to be strings since it would never change on type, no?

Reply:
Yes, sorry, I mixed up things.
I spoke with Ethan.
What I noticed is the SqlGetTypeInfo returns the same data types information, no matter what the connection type_mode is.
Ethan is working on it.